src/aap_eda/settings/default.py

```diff
@@ -442,14 +442,22 @@ def get_rq_queues() -> dict:
 RULEBOOK_QUEUE_NAME = settings.get("RULEBOOK_QUEUE_NAME", "activation")
 
 RQ_STARTUP_JOBS = []
+
+# Id of the scheduler job it's required when we have multiple instances of
+# the scheduler running to avoid duplicate jobs
 RQ_PERIODIC_JOBS = [
     {
         "func": (
             "aap_eda.tasks.orchestrator.enqueue_monitor_rulebook_processes"
         ),
         "interval": 5,
+        "id": "enqueue_monitor_rulebook_processes",
+    },
+    {
+        "func": "aap_eda.tasks.project.monitor_project_tasks",
+        "interval": 30,
+        "id": "monitor_project_tasks",
```

Comment:
Is this newly added `id` used as the parameter for function `monitor_project_tasks`? How does it work?

Reply:
@hsong-rh 
This id is the job_id for rq_redis. Works in the same way as we do in unique_enqueue:
https://github.com/ansible/eda-server/blob/main/src/aap_eda/core/tasking/__init__.py#L312

You can see the function signature here: 
https://github.com/rq/rq-scheduler/blob/master/rq_scheduler/scheduler.py#L241